dvc/stage/__init__.py

```diff
@@ -306,7 +306,10 @@ def remove(self, force=False, remove_outs=True, purge=True):
     @rwlocked(read=["deps"], write=["outs"])
     def reproduce(self, interactive=False, **kwargs):
         if not (kwargs.get("force", False) or self.changed()):
-            logger.info("Stage '%s' didn't change, skipping", self.addressing)
+            if '.dvc' in self.adressing:
```

Comment:
As I've mentioned in https://github.com/iterative/dvc/issues/4233#issuecomment-660721316 , using `.dvc` suffix is not the best approach:
```suggestion
            if not isinstance(stage, PipelineStage) and stage.is_data_source:
```